Packet++/header/TcpLayer.h

```diff
@@ -98,9 +104,66 @@ namespace pcpp
 		TCPOPT_SACK_PERM =      4,
 		/** SACK Block */
 		PCPP_TCPOPT_SACK =      5,
-		/** Echo (obsoleted by option ::PCPP_TCPOPT_TIMESTAMP) */
+		/** Echo (obsoleted by option TcpOptionType::PCPP_TCPOPT_TIMESTAMP) */
+		TCPOPT_ECHO =           6,
+		/** Echo Reply (obsoleted by option TcpOptionType::PCPP_TCPOPT_TIMESTAMP) */
+		TCPOPT_ECHOREPLY =      7,
+		/** TCP Timestamps */
+		PCPP_TCPOPT_TIMESTAMP = 8,
+		/** CC (obsolete) */
+		TCPOPT_CC =             11,
+		/** CC.NEW (obsolete) */
+		TCPOPT_CCNEW =          12,
+		/** CC.ECHO(obsolete) */
+		TCPOPT_CCECHO =         13,
+		/** MD5 Signature Option */
+		TCPOPT_MD5 =            19,
+		/** Multipath TCP */
+		TCPOPT_MPTCP =          0x1e,
+		/** SCPS Capabilities */
+		TCPOPT_SCPS =           20,
+		/** SCPS SNACK */
+		TCPOPT_SNACK =          21,
+		/** SCPS Record Boundary */
+		TCPOPT_RECBOUND =       22,
+		/** SCPS Corruption Experienced */
+		TCPOPT_CORREXP =        23,
+		/** Quick-Start Response */
+		TCPOPT_QS =             27,
+		/** User Timeout Option (also, other known unauthorized use) */
+		TCPOPT_USER_TO =        28,
+		/** RFC3692-style Experiment 1 (also improperly used for shipping products) */
+		TCPOPT_EXP_FD =         0xfd,
+		/** RFC3692-style Experiment 2 (also improperly used for shipping products) */
+		TCPOPT_EXP_FE =         0xfe,
+		/** Riverbed probe option, non IANA registered option number */
+		TCPOPT_RVBD_PROBE =     76,
+		/** Riverbed transparency option, non IANA registered option number */
+		TCPOPT_RVBD_TRPY =      78,
+		/** Unknown option */
+		TCPOPT_Unknown =        255
+	};
+
+	/**
+	 * TCP options types
+	 */
+	enum class TcpOptionEnumType : uint8_t
+	{
+		/** Padding */
+		PCPP_TCPOPT_NOP =       1,
```

Comment:
I think we can get rid of prefix if we are using enum class, e.g. `TcpOptionEnumType::NOP`, not `TcpOptionEnumType::PCPP_TCPOPT_NOP`